### PR TITLE
Ana/add 12-words seeds to seed phrase validation

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

An user requested this since he's got a 12-words seed

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmSessionImport.vue [hunk 1]
```diff
@@ -52,10 +52,6 @@ import SessionFrame from "common/SessionFrame"
 import { mapGetters } from "vuex"
 import Steps from "../../ActionModal/components/Steps"
 
-const words24 = param => {
```
Comment (faboweb) [inline src/components/common/TmSessionImport.vue:55]: have you considered words12 and words24?

Comment (Bitcoinera) [reply src/components/common/TmSessionImport.vue:55]: yes, I was actually thinking about it and making this an "official" PR 🤔 

Will go for it! Thanks 💪 !

**Comment by Bitcoinera** [comment]
Wonder how we didn't have this earlier! The initial sale's seeds are 12-words apparently. We were losing users probably on this

**Comment by faboweb** [comment]
No most of them converted to 24 words at some point as they are using ledgers

**Comment by Bitcoinera** [comment]
Closed in favor of #3883

**Pull request closed without merging**